src/Aks/Aks/Commands/RunAzureRmAksCommand.cs

```diff
@@ -154,7 +154,7 @@ private string GetCommandContext()
                 {
                     foreach (string filepath in filesToAttach)
                     {
-                        var relativePath = filepath.Replace(rootDirectory, "").Trim('/').Trim('\\');
+                        var relativePath = filepath.Replace(rootDirectory, "").Replace('\\', '/').Trim('/');
```

Comment:
Should we use System.IO.Path.DirectorySeparatorChar to meet separator definitions in different OSs? BTW, please add change log for your change.

Reply:
The changelog is updated. Can't update "/" to System.IO.Path.DirectorySeparatorChar because Zip Spec does not support "\" but only "/".